packages/contracts/package.json

```
@@ -4,7 +4,10 @@
   "scripts": {
     "test": "forge test -vvv",
     "test:watch": "forge test -vvv --watch",
-    "start": "anvil & copyfiles ../../.env contracts/.env && forge script script/PublicResolver.s.sol --rpc-url http://localhost:8545 --broadcast -vv",
+    "start": "anvil & copyfiles ../../.env contracts/.env && forge script script/PublicResolver.s.sol --rpc-url http://localhost:8545 --broadcast -vv ",
+    "start:ArbitrumLocally": "copyfiles ../../.env contracts/.env && forge script script/ArbitrumDeployL1.s.sol --rpc-url http://127.0.0.1:8545 --broadcast -vv && forge script script/ArbitrumDeployL2.s.sol --rpc-url http://localhost:8547 --broadcast -vv",
+    "deploy:ArbitrumLayer1": "copyfiles ../../.env contracts/.env && forge script script/ArbitrumDeployL1.s.sol --gas-estimate-multiplier 200 --rpc-url https://sepolia.drpc.org --broadcast -vv",
+    "deploy:ArbitrumLayer2": "copyfiles ../../.env contracts/.env && forge script script/ArbitrumDeployL2.s.sol --gas-estimate-multiplier 700  --rpc-url https://sepolia-rollup.arbitrum.io/rpc --broadcast -vv",
```

this syntax is getting pretty crazy innit? we could improve it by doing something like this:

```md
scripts
├── deploy
│   ├── arbitrum.l1.s.sol
│   ├── arbitrum.l2.s.sol
│   ├── database.s.sol
│   ├── public.s.sol
├── start
│   ├── arbitrum.l1.s.sol
│   ├── arbitrum.l2.s.sol
│   ├── database.s.sol
│   ├── public.s.sol
```

and then changing the script to:

```suggestion
    "start:public": "anvil & copyfiles ../../.env contracts/.env && forge script script/start/public.s.sol --rpc-url http://localhost:8545 --broadcast -vv ",
    "start:arb": "copyfiles ../../.env contracts/.env && forge script script/start/arbitrum.l1.s.sol --rpc-url http://127.0.0.1:8545 --broadcast -vv && forge script script/start/arbitrum.l1.s.sol --rpc-url http://localhost:8547 --broadcast -vv",
    "deploy:arb:l1": "copyfiles ../../.env contracts/.env && forge script script/deploy/arbitrum.l1.s.sol --gas-estimate-multiplier 200 --rpc-url https://sepolia.drpc.org --broadcast -vv",
    "deploy:arb:l2": "copyfiles ../../.env contracts/.env && forge script script/deploy/arbitrum.l2.s.sol --gas-estimate-multiplier 700  --rpc-url https://sepolia-rollup.arbitrum.io/rpc --broadcast -vv",
```